### PR TITLE
Add sprites, costumes, and backdrops from Ipzy

### PR DESCRIPTION
**Pull request by thisandagain**
Adds new sprites, costumes, and backdrops from ST intern: [Ipzy!](https://scratch.mit.edu/users/ipzy/)

### CODE REVIEW

File: src/lib/libraries/backdrops.json [hunk 1]
```diff
@@ -120,6 +120,20 @@
             1
         ]
     },
+    {
+        "name": "Ocean",
+        "md5": "fb907f72b310acc8b95cbf2d2cccabc9.svg",
+        "type": "backdrop",
+        "tags": [
+            "underwater",
+            " ipzy"
```
Comment (thisandagain) [inline src/lib/libraries/backdrops.json]: @carljbowman Don't leave a space between tags. E.g. `[tag,another tag,foo,bar]`

File: src/lib/libraries/costumes.json [hunk 2]
```diff
@@ -276,6 +276,116 @@
             1
         ]
     },
+    {
+        "name": "Bear-a",
+        "md5": "5a4148d7684fc95f38c58a1672062c9e.svg",
+        "type": "costume",
+        "tags": [],
```
Comment (thisandagain) [inline src/lib/libraries/costumes.json]: @carljbowman Did you want to tag the costumes as well?

File: src/lib/libraries/sprites.json [hunk 3]
```diff
@@ -5999,6 +7096,62 @@
             "spriteInfo": {}
         }
     },
+    {
+        "name": "Shark-a",
```
Comment (thisandagain) [inline src/lib/libraries/sprites.json]: @carljbowman I'm guess you meant just "Shark" here.

**Comment by thisandagain** [comment]
@carljbowman Just a few minor notes for you. Let me know when you'd like me to update the PR. 😄

File: src/lib/libraries/sprites.json (hunk 3 above)
Comment (carljbowman) [reply src/lib/libraries/sprites.json]: Good catch. Fixed

File: src/lib/libraries/costumes.json (hunk 2 above)
Comment (carljbowman) [reply src/lib/libraries/costumes.json]: Weird. Those had tags, maybe fixing the spacing issue will resolve this one.

File: src/lib/libraries/backdrops.json (hunk 1 above)
Comment (carljbowman) [reply src/lib/libraries/backdrops.json]: Remove all spaces between tags. Much easier to read these in the JSON then the 2.0 field editor UI.

**Comment by carljbowman** [comment]
@thisandagain - Thanks for the notes. Everything should be updated accordingly.

**Pull request merged**